database/storeDb.js

```diff
@@ -0,0 +1,127 @@
+  var randomWordsArray = randomWords.split(" ")  
+  var returnedRandomString = ""
+  var randomLength = Math.floor((randomNumberGenerator(20))) //Generates a random number between 0 - 20 
+  var randomWord = Math.floor((randomNumberGenerator(341)))
+  for (var i = 0; i < randomLength; i++){
+    var randomWord = Math.floor((randomNumberGenerator(341)))    
+    returnedRandomString += randomWordsArray[randomWord] + " "
+  }
+  return returnedRandomString
+}
+
+var isReturnable = function(){
+  if (Math.floor(randomNumberGenerator(2)) / 2 === 0) {
+    return true;
+  }
+  return false
+}
+
+var randomDescription = function(){
+  var description = ""
+  var randomWordsArray = randomWords.split(" ");    
+  var randomLength = Math.floor(randomNumberGenerator(10) + 15);
+  for (let i = 0; i < randomLength; i++){
+    var randomWord = Math.floor((randomNumberGenerator(341)));    
+    description += randomWordsArray[randomWord] + " ";
+  }  
+  return description;
+}
+
+var randomOverview = function(){
+  var comments = Math.floor(randomNumberGenerator(4))
+  var overview = [];
+  for (let i = 0; i < comments; i++){
+    overview.push(randomPhraseGenerator());
+  }
+  return overview;
+}
+
+var randomOptions = function(){
+  var options = {};  //Storing all of the possible options
+  var randomWordsArray = randomWords.split(" "); //splits the random words into an array 
+  var possibleOptions = Math.floor(randomNumberGenerator(4)) //generates a number of 0 - 3 option possibilities 
+  for (let i = 0; i < possibleOptions; i++){ //Iterates possibilities amount of times 
+    var possibleValues = Math.floor((randomNumberGenerator(341)));  //generates a random between between 0 - 341
+    var keyValue = randomWordsArray[possibleValues];  //The option selection will be this value 
+    var optionSelection = [];  //Array of options being put into the options value
+    var selections = Math.floor(randomNumberGenerator(20)) //Generates x amount of possibilities to select from
+    for (let j = 0; j < selections; j++){ //Iterates through x amount of possibilities 
+      var possibleSelection = Math.floor((randomNumberGenerator(341)));  
+      optionSelection.push(randomWordsArray[possibleSelection]);
+    }
+    options[keyValue] = optionSelection; //Sets the key with x amount of options
+  }
+  return options;
+}
+
+
+
+var checkInventory = function(){
+
+//Filter through the inventory list checking if a specific inventory is in stock
+}
+
+var updateInventory = function(){
+  for (let itemId = 0; itemId < 100; itemId++){
+    itemId = new itemIdnventoryList({
+      productId: itemId, 
+      price: randomNumberGenerator(100),
+      description: randomDescription(), 
+      overview: randomOverview(),
+      options: randomOptions(), //Let's me create an obj
+      quantityInStock: randomNumberGenerator(100),
+      availableToReturn : isReturnable(), 
+      shipping : shippingCost()     
+    });
+    itemId.save(function(err, itemId) => {
```

Comment:
this is a fine implementation but you'll get duplicate records without making the productID a unique key.